gooddata-pandas/gooddata_pandas/dataframe.py

```diff
@@ -229,7 +230,10 @@ def for_insight(self, insight_id: str, auto_index: bool = True) -> pandas.DataFr
         return self.for_items(columns, filter_by=filter_by, auto_index=auto_index)
 
     def for_exec_def(
-        self, exec_def: ExecutionDefinition, label_overrides: LabelOverrides = {}
+        self,
+        exec_def: ExecutionDefinition,
+        label_overrides: LabelOverrides = {},
+        result_size_limits: ResultSize = (),
```

Comment:
def for_exec_def(
        self,
        exec_def: ExecutionDefinition,
        label_overrides: LabelOverrides = None,
        result_size_limits: ResultSize = None,
    ) -> Tuple[pandas.DataFrame, BareExecutionResponse]:
if label_overrides is None:
    label_overrides = {}
if result_size_limits is None:
   result_size_limits = ()

Reply:
I'm not changing label_overrides, as it's not in the scope of my change. Also, why is the original wrong?

Reply:
fixed